spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/autoconfig/AxonServerAutoConfiguration.java

```diff
@@ -96,31 +99,22 @@ public AxonServerConnectionManager platformConnectionManager(AxonServerConfigura
 
     @Bean( destroyMethod = "disconnect")
     @Primary
-    @ConditionalOnMissingBean(CommandBus.class)
-    public AxonServerCommandBus commandBus(TransactionManager txManager,
+    @ConditionalOnMissingQualifiedBean(qualifier = "!unqualified", beanClass = CommandBus.class)
```

Comment:
Curious: Will the qualifier of a bean be `unqualified` if no name is given to it?

Reply:
I should know that, the author tag on that is me :see_no_evil: